core/src/main/java/io/questdb/griffin/engine/functions/json/JsonExtractFunction.java

```diff
@@ -384,6 +366,26 @@ public int getVarcharSize(Record rec) {
 
     @Override
     public void init(SymbolTableSource symbolTableSource, SqlExecutionContext executionContext) throws SqlException {
+        ScalarFunction.super.init(symbolTableSource, executionContext);
+        if (stateA == null) {
```

Comment:
nit: let's add `assert stateB == null;` as the next line to make sure that the field initialization invariant is preserved.